katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -6,19 +6,13 @@
     "properties": {
         "version": {
             "type": "string",
-            "enum": ["1.0", "1.1", "2.0", "2.1", "2.2", "2.3", "2.4", "2.5", "2.6"]
```

Comment:
Any impact to retiring 1.x support ? (I suspect not, but need to ask)

Reply:
Happy with that - if we are going to cull we probably should cull to the latest version...

Reply:
I'll remove pre-2.0 for now. Trying to remove 2.0-2.3 in this PR will probably conflict with another branch I'm working on, but I'll take this as license to drop support for them in a later PR.